docker-compose.yml

```diff
@@ -14,12 +13,14 @@ services:
       - "5000:5000"
     environment:
       REDASH_WEB_WORKERS: 4
+    links:
+      - postgres:postgres
   scheduler:
     <<: *redash-service
     command: scheduler
-    environment:
-      QUEUES: "celery"
-      WORKERS_COUNT: 1
+    # environment:
```

Comment:
doesn't the updated scheduler service need queue name and worker count?

Reply:
No. That’s a change in v10. I followed the upgrade path documented here

https://github.com/getredash/redash/releases/tag/v10.0.0